eZ/Bundle/EzPublishIOBundle/Migration/FileLister/FileRowReader/LegacyStorageFileRowReader.php

```diff
@@ -42,7 +42,7 @@ public function __construct(Connection $connection)
     {
         $row = $this->statement->fetch();
 
-        return $this->prependMimeToPath($row['filename'], $row['mime_type']);
+        return is_array($row) ? $this->prependMimeToPath($row['filename'], $row['mime_type']) : null;
```

Comment:
This should be more strict, the fetch result can be either array or boolean, so this should be enough:

```suggestion
        return false !== $row ? $this->prependMimeToPath($row['filename'], $row['mime_type']) : null;
```